src/queuemanager.cpp

```diff
@@ -105,6 +106,16 @@ std::string QueueManager::generate_enqueue_filename(const std::string& url,
 	fmt.register_fmt('t', title);
 	fmt.register_fmt('e', extension);
 
+	if (feed->rssurl() != item->feedurl() &&
+		item->get_feedptr() != nullptr) {
+		auto feedtitle = utils::replace_all(
+			item->get_feedptr()->title(), "<", "<>");
```

Comment:
Please use `utils::quote_for_stfl` here. Also, `feedtitle` is a constant, and can be marked as such.

Reply:
You got it.

**Sidenote**: there are a couple of similar calls in `ItemListFormAction::item2formatted_line` which might warrant replacing.

Reply:
I won't mind if you add a commit to this PR fixing those lines in `ItemListFormAction`. Even though it's not directly related to the purpose of this PR, it's a small, well-understood, and uncontroversial change that can be justified by boyscout's rule.